Migrations/Version_0_0_22.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\CustomObjectsBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\IntegrationsBundle\Migration\AbstractMigration;
+
+class Version_0_0_22 extends AbstractMigration
+{
+    private string $table = 'custom_item_export_scheduler';
+
+    private Schema $schema;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function isApplicable(Schema $schema): bool
+    {
+        $this->schema = $schema;
+
+        return !$schema->hasTable($this->concatPrefix($this->table));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function up(): void
```

Comment:
I am use to using doctrine+maker for this which I am guessing we don't do since this extends a Mautic class.

Do we not support downgrades as well? If we do, id suggest we always do it.